.github/workflows/lxd-ci.yml

```diff
@@ -0,0 +1,79 @@
+name: LXD-CI
+
+on:
+  workflow_dispatch:
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Clone this repository
+      uses: actions/checkout@v2
+
+    - name: Set LXD package envs
+      run: |
+        echo "LXD_PACKAGE_VER=$(head -n1 debian/changelog | grep -oP '(?<=lxd\ \().*(?=\)\ UNRELEASED)')" >> $GITHUB_ENV
+        echo "LXD_PACKAGE_ARCH=$(cat debian/control | grep -oP '(?<=Architecture\:\ ).*(?=$)')" >> $GITHUB_ENV
+
+    - name: Build LXD package
+      run: make build
+
+    - name: Upload LXD package
+      uses: actions/upload-artifact@v2.2.3
+      with:
+        name: lxd_${{ env.LXD_PACKAGE_VER }}_${{ env.LXD_PACKAGE_ARCH }}.deb
+        path: lxd_${{ env.LXD_PACKAGE_VER }}_${{ env.LXD_PACKAGE_ARCH }}.deb
+
+  testing:
```

Comment:
Lest remove testing, for now focus on automating DEB builds

Reply:
Yes, I agree with you.
Removed this section from workflow.